src/mainPage/Header.tsx

```diff
@@ -1,17 +1,124 @@
-import reactLogo from "../assets/react.svg";
-import Navbar from "./Navbar";
+import styled from "styled-components";
+import logoNoBackground from "../assets/png/logoNoBackground.png";
+import ToggleTheme from "../menu/ToggleTheme";
+import Navbar from "../menu/Navbar";
+import { useState } from "react";
 
 const Header = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const toggleMenu = () => {
+    if (isMenuOpen == false) {
+      setIsMenuOpen(true);
+    } else {
+      setIsMenuOpen(false);
+    }
+  };
+  console.log(isMenuOpen);
```

Comment:
Remove the `console.log`. It's not a good idea to commit console.log for test purposes.